src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -251,10 +250,8 @@ def get_env(self):
 
         # Do not use the standard .log file extension, otherwise it'll get
         # removed (cleaned up for disk space savings) before we can archive it.
-        log_filename = test_name + "_" + \
-                       env_CCSA + "_" + \
-                       env_PHY_ADDR_STR + "_" + \
-                       "output.orterun_log"
+        log_filename = test_name + "_" + env_CCSA + "_" + env_PHY_ADDR_STR + "_" + \
+            "output.orterun_log"
```

Comment:
If you wanted to be fancy
```suggestion
        log_filename = '_'.join([test_name, env_CCSA, env_PHY_ADDR_STR, "output.orterun_log"])
```